config/environments/production.rb

```diff
@@ -16,7 +16,7 @@
   # config.action_dispatch.rack_cache = true
 
   # Disable Rails's static asset server (Apache or nginx will already do this).
-  config.public_file_server.enabled = false
+  config.public_file_server.enabled = ENV.fetch('RAILS_SERVE_STATIC_FILES', false) == 'true'
```

Comment:
This should be documented in https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/manuals/nightly/3.4_install_from_source.md